Makefile

```diff
@@ -66,3 +75,10 @@ image-release:
 .PHONY: push-release
 push-release:
 	docker push $(IMAGE):$(VERSION)
+
+generate-kustomize: bin/helm
+	cd charts/aws-fsx-csi-driver && ../../bin/helm template kustomize . -s templates/csidriver.yaml > ../../deploy/kubernetes/base/csidriver.yaml
```

Comment:
Is `helm template kustomize` a command?  A cursory search by me didn't find a good reference.